emergence_lib/Cargo.toml

```diff
@@ -15,7 +15,7 @@ debug_tools = { path = "../tools/debug" }
 rand = "0.8"
 leafwing-input-manager = "0.7"
 emergence_macros = { path = "../emergence_macros" }
-bevy-trait-query = "0.1"
+bevy-trait-query = { git = "https://github.com/Leafwing-Studios/bevy-trait-query", branch = "added-changed" }
```

Comment:
I'd recommend to specify a commit hash to make installs more reliable

Reply:
Yes!